master/getting-started/kubernetes/installation/app-layer-policy.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Installing Calico with Application Layer Policy
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/app-layer-policy'
+---
+
+Application Layer Policy for {{site.prodname}} allows you to write policies that enforce against both network layer attributes, like IP addresses or ports and application layer attributes like HTTP Method or Path. It can also enforce policy based on cryptographically secure identities, allowing you to reduce trust in your network.  See [Application Layer Policy Introduction](link me) for more information.
+
+# Kubernetes Cluster Requirements
+
+Application Layer Policy is only supported on Kubernetes, and the cluster must meet the following requirements:
+
+  * Kubernetes v1.9 or later
+  * [Mutating Webhook](https://kubernetes.io/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19) support is enabled
+
+# Enabling support in {{site.prodname}}
```

Comment:
I would suggest perhaps making this title a little more descriptive, specific, and informative...perhaps something like "Enabling {{site.prodname}} application layer policy"?

As for the section below, I would recommend instead of asking them to go to another section and just replace that YAML name with this one, to give them a set of `kubectl apply -f` links and tell them to use the one that corresponds to their installation method. This will probably be much easier for people to follow along, as a more stepwise-type procedure. You might also preface this with a discussion of how the `calico/node` images will restart.